CHANGES.md

```diff
@@ -1 +1,48 @@
-## DC/OS 1.12.0
\ No newline at end of file
+## DC/OS 1.12.0
+
+```
+* For any significant improvement to DC/OS add an entry to Fixed and Improved section.
+* For Security updates, please call out in Security updates section.
+* Add to the top of the existing list.
+* External Projects like Mesos and Marathon shall provide a link to their published changelogs.
+
+Format of the entries must be.
+
+* Entry with no-newlines. (DCOS_OSS_JIRA)
+<new-line>
+* Entry two with no-newlines. (DCOS_OSS_JIRA_2)
+```
+
+
+### Fixed and improved
+
+* Windows-based pkgpanda builds (DCOS_OSS-1899)
+
+* Upgrade REX-Ray to v0.11.2. (DCOS_OSS-2245)
+
+* Moves the dcos-metrics Prometheus producer from port 9273 to port 61091. Using a port which is not available to Mesos (any unoccupied port > 61000) avoids the need to update anything. (DCOS-21594)
```

Comment:
The issue number leads me to an issue which seems unrelated to the text.

Reply:
(I'd like to work on the text, but dunno which issue to look up.)

Reply:
- DC/OS Metrics: moved the ...

Reply:
Addressed this comment.  Included the correct JIRA ID.